Source/WebCore/platform/graphics/SourceBrush.h

```diff
@@ -33,19 +33,14 @@ namespace WebCore {
 
 class SourceBrush {
 public:
+    
+    struct LogicalGradient {
+        Ref<Gradient> gradient;
+        AffineTransform spaceTransform;
+    };
+    
     struct Brush {
-        struct LogicalGradient {
```

Comment:
I don't understand why you moved this but it remains a nested struct.  I think we should leave it.